src/asm/lexer.c

```diff
@@ -1897,11 +1897,16 @@ static int yylex_NORMAL(void)
 				return T_OP_LOGICGE;
 			case '>':
 				shiftChar();
-				if (peek() == '=') {
+				switch (peek()) {
```

Comment:
Update the comment above to `/* Either >>=, GT, GTE, or right shift signed or unsigned */